File: server/user-service/src/main/java/com/sudoo/userservice/controller/dto/AddressDto.java
package com.sudoo.userservice.controller.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AddressDto {
    private String addressId;

    private int provinceID;

    private int districtID;

    private String wardCode;

    private String provinceName;

    private String districtName;

    private String wardName;

    private String address;

    private String fullAddress;
}

File: server/auth-service/src/main/kotlin/com/sudoo/authservice/dto/AddressDto.kt
package com.sudoo.authservice.dto

data class AddressDto(
    private var addressId: String? = "",

    val provinceID: Int = 0,

    val districtID: Int = 0,

    val wardCode: Int = 0,

    val provinceName: String = "",

    val districtName: String = "",

    val wardName: String = "",

    val address: String = "",

    val location: LocationDto = LocationDto(),

    val fullAddress: String = "",
)